sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -146,6 +153,32 @@ export class StudentController extends BaseController {
     );
   }
 
+  /**
+   * This returns only a part of the EducationProgram details for the student
+   * @param programId
+   * @returns
+   */
+  @Get("/education-program/:programId/summary")
```

Comment:
Do we need to have the "_summary_" appended here? I believe that this could be reused anytime that there is a need to retrieve the program data for a student.
Following this, the property "_credentialType_" on DTO could be renamed to "_credentialTypeToDisplay_" allowing the DTO in the future to return the real "_credentialType_" from the database.

Reply:
removed the /summary